scripts/populate_preprint_providers.py

```diff
@@ -73,6 +73,12 @@ def update_or_create(provider_data):
             new_provider.default_license = get_license(default_license)
             new_provider.save()
         provider = PreprintProvider.load(new_provider._id)
+        # add NotificationSubscription instances to provider
+        NotificationSubscription.objects.get_or_create(
```

Comment:
Does this need to be here? Shouldn't it be either
A) added in migration `0085` if data already exists
B) handled by the post-save handler if data doesn't already exist?